tests/Common/AnotherUnit.h

```diff
@@ -3,6 +3,8 @@
 // Verifies the header is included when the delegate is defined in a different file
 typedef void (*DelegateInAnotherUnit)();
 
+struct DuplicateStruct;
```

Comment:
Add a function which uses this struct and call that function in the test code. We must ensure such functions are generated.